server/routes/appointments/create-and-edit/handlers/dateAndTime.ts

```diff
@@ -41,6 +42,12 @@ export default class DateAndTimeRoutes {
     res.render('pages/appointments/create-and-edit/date-and-time', {
       backLinkHref: getAppointmentBackLinkHref(req, 'location'),
     })
+    const properties = {
+      username: res.locals.user.username,
+      prisonCode: res.locals.user.activeCaseLoadId,
+      property: 'Date and Time',
+    }
+    trackEvent('SAA-Appointments-Appointment-Change-From-Schedule', properties, null)
```

Comment:
This will track this event every time the date and time page is navigated to including the initial run through of the journey and from check answers. The requirement was to track the event when a user clicks on a change link from the schedule page.

This could be done in a few ways. Could be a new `CHANGE` function in ScheduleRoutes which takes a property (the url fragment), tracks that property and then redirects the user to that page. It could check the referer although I don't think that's always accurate or it could be a query string parameter.

Reply:
done